core/src/main/resources/AppConstants.properties

```diff
@@ -361,7 +361,6 @@ msg.log.hidden.default=true
 msg.log.keys=conversationId
 # Define xPath for key conversationId
 msg.log.xPath.conversationId=*[local-name()='Envelope']/*[local-name()='Header']/*[local-name()='MessageHeader']/*[local-name()='HeaderFields']/*[local-name()='ConversationId']
-msg.log.xPath.value=/test/value
```

Comment:
The test-value should also be added to the file "DeploymentSpecifics.properties" in the iaf-test project resources. Otherwise the test will not fully run the intended code path and not actually test the changes.

Reply:
Ah ok, will do.